packages/fxa-settings/src/components/Settings/AccountRecoveryWizard/index.tsx

```diff
@@ -0,0 +1,133 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+
+export const AccountRecoveryWizard = (props: RouteComponentProps) => {
+  const [currentStep, setCurrentStep] = useState(1);
+  /*
+    Making this a component (as opposed to writing it directly into the PageRecoveryAdd view) will hopefully allow us to separate the logic a little more from the UI.
+    In this wizard pattern, you'll want to have equality checks which control which step is being displayed (like,
+      { currentStep === 1 && (<YourViewHere/> )}
+    ).
+    Each step will be built with the <FlowContainer /> component as the outer wrapper.  Each step will at minimum expect the props `currentStep` and `setCurrentStep`, received from this component. The `currentStep` value and `setCurrentStep` method will be passed in to the `<FlowContainer/>` as props of the same name. Each step will want to have the <ProgressBar /> component as a child, and the title will be 'Account recovery key'.
+    We have filler components hardcoded in here (the Wizard pattern seems too simple to justify making it into a reusable component), but we can replace these as we create the real views.
+    Each step should be a separate component.
+  */
+  return (
+    <>
+      {
+        // Create an account recovery key in case you forget your password
+        currentStep === 1 && (
+          <FlowContainer
+            title="Account recovery key"
+            {...{ currentStep, setCurrentStep }}
+            onBackButtonClick={() => {
+              window.history.back();
+            }}
+          >
+            <ProgressBar currentStep={1} numberOfSteps={4} />
```

Comment:
Since we know we're in `currentStep === 1` (per L25), should these just be something like:
```html
<ProgressBar numberOfSteps={4} {...{ currentStep }} />
```

Although now `numberOfSteps` is basically a `const` since it presumably won't change mid-wizard, so maybe gets hoisted up to a parent scoped `const numberOfSteps = 4;` and this is:
```html
<ProgressBar {...{ currentStep, numberOfSteps }} />
```

Reply:
I tried to cover this in the comment within the file but I think my logic is just too muddy -- these individual views are going to be replaced by components, so this is all just filler so that we can skip through the sequence as we put those views together.